src/main/java/sirius/kernel/commons/Outcall.java

```diff
@@ -187,9 +198,35 @@ public Outcall(URI uri) {
      * @return the underlying {@link HttpClient.Builder}
      */
     public HttpClient.Builder modifyClient() {
+        return modifyClient(null);
```

Comment:
Vielleicht wäre es eleganter den clientSelector einmalig im Constructor zu setzen anstatt ihn immer mitschleifen zu müssen?

Reply:
The selector is _only_ required if the builder is customized. For many many cases, the single central http client can be re-used. Therefore, the constructor was left untouched and the selector has been moved here.

The few cases where we customize the builder are always either: modify timeout, provide a cookie manager or accept self-signed certificates.

Especially with cookie managers we want to enforce a decision if the resulting client can be cached or not and therefore require a parameter here.